xarray/core/dataarray.py

```diff
@@ -1220,6 +1220,91 @@ def fillna(self, value):
         out = ops.fillna(self, value)
         return out
 
+    def interpolate_na(self, dim=None, method='linear', limit=None,
+                       use_coordinate=True,
+                       **kwargs):
+        """Interpolate values according to different methods.
+
+        Parameters
+        ----------
+        dim : str
+            Specifies the dimension along which to interpolate.
+        method : {'linear', 'time', 'index', 'values', 'nearest'}, optional
```

Comment:
Should probably some explanation about `method='time'` option is necessary here.

Reply:
good catch. we actually dropped the time/index/values methods in favor of a more explicit interface that uses `use_coordinate`.